package.json

```diff
@@ -17,7 +17,8 @@
     "server-dev": "nodemon --exec 'ts-node --files' src/server/index.ts",
     "docker-dev": "concurrently \"npm run server-dev\" \"npm run client-dev\"",
     "dev": "docker-compose -f docker-compose.yml up --build",
-    "test": "jest --collectCoverage"
+    "test": "jest --collectCoverage",
+    "coveralls": "jest --coverage && coveralls < coverage/lcov.info"
```

Comment:
can we simply fold `coveralls` into existing `test` command?

Reply:
Seems redundant to send coverage reports for coveralls when running tests locally though

Reply:
suggest to rename the command to `test:ci` in that case to convey clarity about what it does and who is supposed to run the command